### ISSUE TITLE
is there a way to disable Apple pay button from appearing 

### ISSUE THREAD

**Issue by elabda3company** [Tap-Payments/goSellSDK-iOS#51]
is there a way to disable Apple pay button from appearing 
or completely disable apple pay ?

**Comment by hexalitics** [comment]
@elabda3company have you got the solution ?

**Comment by elabda3company** [comment]
> [@elabda3company](https://github.com/elabda3company) have you got the solution ?

yes it was old issue and fixed you can disable it from the code directly
